base/mmcif_ma-core.dic

```diff
@@ -6043,6 +7058,8 @@ save__ma_entry_associated_files.file_format
     'tsv'
     'pdf'
     'json'
+    'png'
+    'jpg'
```

Comment:
The change above was required for `_ma_associated_archive_file_details.file_format`. I would suggest to merge the two `file_format` enums and have the exact same enum in both categories (gzip and zip are to be added to `_ma_associated_archive_file_details.file_format` and then that enum is to be used for both).